src/main/java/ca/corefacility/bioinformatics/irida/model/sequenceFile/SequenceFile.java

```diff
@@ -56,10 +37,15 @@
  * particular {@link Sample}.
  */
 @Entity
+@Inheritance(strategy=InheritanceType.SINGLE_TABLE)
+@DiscriminatorColumn(
```

Comment:
We haven't been using the `@DiscriminatorColumn` in our other inheritance strategies.  If you just leave it off, it automatically creates a `DTYPE` column with the class name`.

I'll leave it to you whether to use the `SINGLE_TABLE` or `JOINED` approach.  In general we've preferred the `JOINED` approach because it means less unnecessary blank columns.  If it's just 1 or 2 empty columns it's not a big deal though.

Reply:
I tried it out without the @DiscriminatorColumn and it throws the following error:
`The column name DTYPE was not found in this ResultSet`

For this case it's probably best to use a `SINGLE_TABLE` approach as there are no extra fields being declared in the LocalSequenceFile or CloudSequenceFile classes

Reply:
Updated in e9d5d85b90d82ec70a6779780f6b0d0fd138cd82